client/package.json

```diff
@@ -3,32 +3,33 @@
     "version": "0.0.0",
     "scripts": {
         "ng": "ng",
-        "start": "ng serve --proxy-config proxy.conf.json --host=0.0.0.0 --aot",
-        "build": "ng build",
+        "start": "ng serve --proxy-config proxy.conf.json --host=0.0.0.0",
+        "build": "ng build --prod --aot",
```

Comment:
shouldn't we have a build for _normal_ build and one for _prod_?

Reply:
you can use ng build if you have to. this build command was run using `npm run build`